README.md

```diff
@@ -22,3 +22,182 @@ Right now there are a lot of scripts, in many languages, and even container imag
 We hope to make `falcoctl` the source of truth for these tasks or chores and give operators a first class experience.
 
 Recently, we started an effort to revamp this project and make it a first-class citizen in the Falco ecosystem. As the first step, we are currently working on implementing a [proposal](proposals/20220916-rules-and-plugin-distribution.md) to allow our users to consume and install distributed plugins and rules files easily.
+
+## Installation
+### Install falcoctl manually
+You can download and install *falcoctl* manually following the appropriate instructions based on your operating system architecture.
+#### Linux
+##### AMD64
+```bash
+curl --fail -LS "https://github.com/falcosecurity/falcoctl/releases/download/v0.2.0-rc1/falcoctl_0.2.0-rc1_linux_amd64.tar.gz" | tar -xz
+sudo install -o root -g root -m 0755 falcoctl /usr/local/bin/falcoctl
+```
+##### ARM64
+```bash
+curl --fail -LS "https://github.com/falcosecurity/falcoctl/releases/download/v0.2.0-rc1/falcoctl_0.2.0-rc1_linux_arm64.tar.gz" | tar -xz
+sudo install -o root -g root -m 0755 falcoctl /usr/local/bin/falcoctl
+```
+> NOTE: Make sure */usr/local/bin* is in your PATH environment variable.
+
+#### MacOS
+##### Intel
+```bash
+curl --fail -LS "https://github.com/falcosecurity/falcoctl/releases/download/v0.2.0-rc1/falcoctl_0.2.0-rc1_darwin_amd64.tar.gz" | tar -xz
+chmod +x falcoctl
+sudo mv falcoctl /usr/local/bin/falcoctl
+```
+##### Apple Silicon
+```bash
+curl --fail -LS "https://github.com/falcosecurity/falcoctl/releases/download/v0.2.0-rc1/falcoctl_0.2.0-rc1_darwin_arm64.tar.gz" | tar -xz
+chmod +x falcoctl
+sudo mv falcoctl /usr/local/bin/falcoctl
+```
+#### Windows
+```bash
+curl --fail -LS "https://github.com/falcosecurity/falcoctl/releases/download/v0.2.0-rc1/falcoctl_0.2.0-rc1_windows_amd64.tar.gz" | tar -xz
+```
+And move it to a file location in your system **PATH**
```

Comment:
Let's remove this for now :) Although I really appreaciate the fact that there is a Windows build already available